examples/rh-sso/rockspec

```diff
@@ -0,0 +1,16 @@
+package = "apicast"
+source = { url = '.' }
+version = '0.1-0'
+dependencies = {
+  'lua-resty-http == 0.09-0',
+  'inspect == 3.1.0-1',
+  'router == 2.1-0',
+  'lua-resty-jwt == 0.1.9-0',
```

Comment:
`lua-resty-jwt` belongs to ` apicast/apicast-0.1-0.rockspec` because it is part of the main codebase.